### DISCUSSION TITLE
Troubleshooting  CP210x USB to UART VCP driver

### DISCUSSION THREAD

**Discussion by bap293** [Moddable-OpenSource/moddable#688] [Getting started] [answered]
I am using windows 10.  Is this the driver I should be using?  CP210x VCP Windows 

If not, which one?

I don't see the any virtual com ports from SI labs.

How does one trouble shoot this driver

**Comment by bap293** [comment, marked as answer]
It was the USB cable, of all things. Had to try 3 cables before I found one that was not data only.  Learned something at least.

**Reply by phoddie** [reply]
Yea, that happens. Congrats on figuring it out. Thanks for letting us know, so we don't put more attention into this one.
